content/administration/maintain/odoo_online.rst

```diff
@@ -35,22 +37,22 @@ Several actions are available:
 Upgrade
 =======
 
-If you are *not* on the latest **Online version**, you should receive an invitation to :doc:`upgrade
-<../upgrade>` your database. A **Rolling Release button** on your database's main screen proposes an
-upgrade to the latest version (e.g., 13.0 to 15.1).
+If the database is *not* on the latest **Online version**, the administrator should receive an
+invitation to :doc:`upgrade <../upgrade>` the database. A **Rolling Release button** on the
+database's main screen proposes an upgrade to the latest version (e.g., 14.0 to 16.1).
 
 .. important::
-   - | **If your Odoo database's version is lower than the latest major release:**
-     | You must upgrade your database within two months. After these two months, an automatic
+   - | **If the Odoo database's version is lower than the latest major release:**
+     | The database must be upgraded within two months. After these two months, an automatic
        upgrade is initiated.
-   - | **If your Odoo database's version is equal to or higher than the latest major release:**
-     | You can disregard the invitation to upgrade as you probably wouldn't benefit from new
+   - | **If the Odoo database's version is equal to or higher than the latest major release:**
+     | Disregard the invitation to upgrade as the database probably wouldn't benefit from new
        features every two months.
 
 .. note::
-   Versions that are not supported anymore become deprecated and need to be updated to avoid
-   security issues. We recommend you initiate the upgrade of the database yourself, as this method
-   allows you to request a test upgrade of your database to check for any discrepancies.
+   Versions that are not supported anymore become deprecated and need to be updated to avoid security
+   issues. Odoo recommends that the company initiate the upgrade, as this method allows for the
+   company to request a test upgrade of the database to check for any discrepancies.
```

Comment:
```suggestion
   Versions that are not supported anymore become deprecated and need to be updated to avoid
   security issues. Odoo recommends that the company initiate the upgrade, as this method allows for
   the company to request a test upgrade of the database to check for any discrepancies.
```